packages/reports/addon/templates/components/common-actions/schedule.hbs

```diff
@@ -81,6 +81,17 @@
           {{/power-select}}
         </fieldset>
       </div>
+      {{#if (feature-flag "enableNotifyWhenEmpty")}}
+        <div class="schedule-modal__section-group">
+          {{x-toggle
+            theme="ios"
```

Comment:
Can we remove this to match the other toggles

Reply:
Can we add some margin around this component it seems a little close to the select above

Reply:
cool. I am not an apple fan tbh XD